test/parallel/test-tls-passphrase.js

```diff
@@ -11,11 +11,17 @@ var tls = require('tls');
 var fs = require('fs');
 var path = require('path');
 
-var key = fs.readFileSync(path.join(common.fixturesDir, 'pass-key.pem'));
+var passKey = fs.readFileSync(path.join(common.fixturesDir, 'pass-key.pem'));
```

Comment:
Nit: While in here anyway, maybe take an opportunity to var->const some stuff? Up to you.